libs/models/sections/projects/src/lib/models-sections-projects.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
```

Comment:
Since this is a models module it doesn't need this file. (it has no declarations to manage)